src/common/function/src/system/version.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::sync::Arc;
+use std::{env, fmt};
+
+use common_query::error::Result;
+use common_query::prelude::{Signature, Volatility};
+use datatypes::data_type::ConcreteDataType;
+use datatypes::vectors::{StringVector, VectorRef};
+
+use crate::function::{Function, FunctionContext};
+
+#[derive(Clone, Debug, Default)]
+pub(crate) struct VersionFunction;
+
+impl fmt::Display for VersionFunction {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "VERSION")
+    }
+}
+
+impl Function for VersionFunction {
+    fn name(&self) -> &str {
+        "version"
+    }
+
+    fn return_type(&self, _input_types: &[ConcreteDataType]) -> Result<ConcreteDataType> {
+        Ok(ConcreteDataType::string_datatype())
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::exact(vec![], Volatility::Immutable)
+    }
+
+    fn eval(&self, _func_ctx: FunctionContext, _columns: &[VectorRef]) -> Result<VectorRef> {
+        let result = StringVector::from(vec![format!(
+            "5.7.20-greptimedb-{}",
```

Comment:
Is 5.7.20 a MySQL version? It doesn't seems a common function but specific to MySQL query.

Reply:
Also, if we don't want full compatible with MySQL, maybe just use `8.0.36` to prevent further imcompatible.

Reply:
> select version() is available in both mysql and PostgreSQL

Yeah. By "not a common function" I mean its manner is for MySQL specifically. But if PG doesn't use this value, it can be fine.

Could you point out where we return the "version header" in PG server call path?